configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml

```diff
@@ -0,0 +1,69 @@
+model:
```

Comment:
Please rename this file as you're not using t5xl for conditioning

Reply:
maybe using fairscale-uncond

Reply:
Sure, thanks